src/bptl/work_units/kownsl/tasks.py

```diff
@@ -123,3 +146,112 @@ def get_approval_status(task: BaseTask) -> dict:
             "num_rejected": num_rejected,
         },
     }
+
+
+@register
+def get_review_response_status(task: BaseTask) -> dict:
+    """
+    Get the reviewers who have not yet responded to a review request so that
+    a reminder email can be sent to them if they exist.
+
+    In the task binding, the service with alias ``kownsl`` must be connected, so that
+    this task knows which endpoints to contact.
+
+    **Required process variables**
+
+    * ``reviewRequestId``: the identifier of the Kowns review request, used to update
+      the object in the API.
+    * ``hoofdZaakUrl``: URL reference to the zaak used for the review itself.
+    * ``kownslUsers``: list of usernames that have been configured in the review request configuration.
+
+    **Sets the process variables**
+
+    * ``remindThese``: a JSON-object containing a list of usernames who need reminding:
+
+      .. code-block:: json
+            [
+                "user1",
+                "user2",
+            ]
```

Comment:
```suggestion
      .. code-block:: json

            [
                "user1",
                "user2",
            ]
```
code-formatting in RST is iffy